src/index.js

```diff
@@ -335,7 +318,7 @@ analytics.subscribe("product_viewed", (event) => {
         event.data,
         productViewedEventMapping,
     );
-    const contextualPayload = mapContextLineObjectKeys(
```

Comment:
what's the requirement here ?

Reply:
We will keep the context object for the event as-is that is generated by the underlying JS SDK, and only do two changes:
1. add `context.shopifyDetails`
2. for the page related properties, JS SDK is catching values from iframe which are incorrect, hence we map them to apt values in the custom pixel payload's own context.